neo/app/assets/javascripts/common/services.js

```diff
@@ -0,0 +1,69 @@
+angular.module('cloudberry.common', [])
+  .service('Asterix', function($http, $timeout) {
```

Comment:
I'm not very clear about the difference between these two. In most of the cases, the service seems used more often for the shared utility functions. So I choose to make it as a service.
Do you know any drawback of using service?